### PR TITLE
[AT360] Integrate IG chart into app

### PR DESCRIPTION
**Pull request by gmzjuliana**
### Purpose
using mock data, add the IG chart into the app

### CODE REVIEW

**Comment by gmzjuliana** [comment]
Heya @federicoweber :) an extra question on this one.. the tests for the reducer won't pass because of the mocks, what would you recommend for when we merge this one in?

File: packages/shared-components/CommonChart/components/ChartTooltip/index.jsx [hunk 1]
```diff
@@ -106,15 +106,15 @@ const PresetsTooltip = ({
   metricData,
   presetConfig,
   metrics,
-}) => (
+}, wording = null) => (
```
Comment (federicoweber) [inline packages/shared-components/CommonChart/components/ChartTooltip/index.jsx]: I think we can remove this right?
It wasn't behaving as expected, and I think we only need to pass `wording` down **as a prop** for the custom tooltip.

File: packages/stories-metrics/components/InfoLabelSelect/story.jsx [hunk 2]
```diff
@@ -11,7 +11,7 @@ const metricsWithClickHandlers = metrics.map((metric, index) => ({
     onItemClick: () => console.info(metric.label, 'Clicked'),
 }));
 
-const selectedMetric = {
+const displayedMetric = {
```
Comment (federicoweber) [inline packages/stories-metrics/components/InfoLabelSelect/story.jsx]: ❤️ good call on renaming this

File: packages/shared-components/CommonChart/components/ChartTooltip/index.jsx (hunk 1 above)
Comment (gmzjuliana) [reply packages/shared-components/CommonChart/components/ChartTooltip/index.jsx]: yup, thanks for catching this

File: packages/server/rpc/index.js [hunk 3]
```diff
@@ -69,7 +70,7 @@ module.exports = rpc(
   getAccountDetails,
   hashtags,
   aggregate,
-  lastUpdated,
+  //lastUpdated,
```
Comment (federicoweber) [inline packages/server/rpc/index.js]: Hey @gmzjuliana I think we removed this, any reason it has been commented in here?

File: packages/shared-components/CommonChart/chartConfig.jsx [hunk 4]
```diff
@@ -125,9 +125,10 @@ export default () => ({
   xAxis: getXAxis(),
   yAxis: getYAxis(),
   chart: {
-    spacingLeft: 15,
-    marginRight: 55,
+    spacingLeft: 24,
+    spacingRight: 24,
     spacingTop: 40,
+    height: 450
```
Comment (federicoweber) [inline packages/shared-components/CommonChart/chartConfig.jsx]: 🙌

File: packages/shared-components/TruncatedNumber/index.jsx [hunk 5]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import numeral from 'numeral';
 
-const TruncatedNumber = ({ children, absoluteValue, shorterOption, showPercentSign }) => {
+const TruncatedNumber = ({ children, absoluteValue, shorterOption, showPercentSign, showTwoDecimals }) => {
```
Comment (federicoweber) [inline packages/shared-components/TruncatedNumber/index.jsx]: Hey @gmzjuliana how do you feel about always showing two decimals when showing percentages?

File: packages/stories-daily/components/StoriesDaily/index.jsx [hunk 6]
```diff
@@ -0,0 +1,223 @@
...
+
+  dailyWithColor() {
+    const { daily, selectedMetrics } = this.props;
+    return daily.map(item => ({
+      day: item.day.toString(),
+      metrics: item.metrics.map(metric => ({
+        ...metric,
+        color:
+          metric.label === selectedMetrics.primary.label
+            ? twitter
```
Comment (federicoweber) [inline packages/stories-daily/components/StoriesDaily/index.jsx]: it took me a bit to realize this was a color 😄

File: packages/stories-daily/middleware.js [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+};
+
+const formatDataForCSVExport = (
+  { selectedMetrics, daily },
+  { profiles, selectedProfileId }
+) => {
+  const data = {
+    date: [],
+    [selectedMetrics.primary.label]: [],
+    [selectedMetrics.secondary.label]: []
```
Comment (federicoweber) [inline packages/stories-daily/middleware.js]: 🤔 might be worth returning all the metrics for a given day. What do you think?

File: packages/server/rpc/index.js (hunk 3 above)
Comment (gmzjuliana) [reply packages/server/rpc/index.js]: sorry just noticed your comments 🙈 I comment this out so I can work without the hot reloader in my local but I just put it back in.. is this meant to be taken out completely?

File: packages/stories-daily/middleware.js (hunk 7 above)
Comment (gmzjuliana) [reply packages/stories-daily/middleware.js]: I kinda like keeping it true to what the chart is showing 🙈

File: packages/shared-components/TruncatedNumber/index.jsx (hunk 5 above)
Comment (gmzjuliana) [reply packages/shared-components/TruncatedNumber/index.jsx]: into it! that's a great way to simplify this :)

**Pull request merged**